Task2/src/app/weatherList/list.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, Input, Output, OnChanges, EventEmitter } from '@angular/core';
```

Comment:
ok. Just try to think about the idea that you will be probably need to implement the same pagination logic? for another part of your application.